code/modules/clothing/suits/reactive_armour.dm

```diff
@@ -5,47 +5,48 @@
 	icon = 'icons/obj/clothing/suits.dmi'
 	w_class = WEIGHT_CLASS_BULKY
 
-/obj/item/reactive_armour_shell/attackby(obj/item/I, mob/user, params)
+/obj/item/reactive_armour_shell/attackby(obj/item/weapon, mob/user, params)
 	..()
 	var/static/list/anomaly_armour_types = list(
 		/obj/effect/anomaly/grav = /obj/item/clothing/suit/armor/reactive/repulse,
 		/obj/effect/anomaly/flux = /obj/item/clothing/suit/armor/reactive/tesla,
 		/obj/effect/anomaly/bluespace = /obj/item/clothing/suit/armor/reactive/teleport,
 		)
 
-	if(istype(I, /obj/item/assembly/signaler/anomaly))
-		var/obj/item/assembly/signaler/anomaly/A = I
-		var/armour_path = anomaly_armour_types[A.anomaly_type]
+	if(istype(weapon, /obj/item/assembly/signaler/anomaly))
+		var/obj/item/assembly/signaler/anomaly/anomaly = weapon
+		var/armour_path = anomaly_armour_types[anomaly.anomaly_type]
 		if(!armour_path)
 			armour_path = /obj/item/clothing/suit/armor/reactive/stealth //Lets not cheat the player if an anomaly type doesnt have its own armour coded
-		to_chat(user, span_notice("You insert [A] into the chest plate, and the armour gently hums to life."))
+		to_chat(user, span_notice("You insert [anomaly] into the chest plate, and the armour gently hums to life."))
 		new armour_path(get_turf(src))
 		qdel(src)
-		qdel(A)
+		qdel(anomaly)
 
-//Reactive armor
+//Reactive armour
 /obj/item/clothing/suit/armor/reactive
-	name = "reactive armor"
+	name = "reactive armour"
```

Comment:
https://github.com/tgstation/tgstation/blob/master/.github/guides/STYLE.md#things-that-do-not-matter

There's zero reason to change all of this it from `armor` to `armour`

```
The following coding styles are not only not enforced at all, but are generally frowned upon to change for little to no reason:

    - English/British spelling on var/proc names 
```